<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GCDA4E4" failures="4" tests="10" name="User Tests" time="0.46" errors="0" timestamp="2020-03-27T17:23:50 IST">
  <testcase classname="com.test.Test1" name="example1" time="0.109">
    <failure type="java.lang.AssertionError" message="This test should be failed expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: This test should be failed expected [3] but found [2]
at com.test.Test1.example1(Test1.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- example1 -->
  <testcase classname="com.test.Test1" name="example2" time="0.002"/>
  <testcase classname="com.test.Test1" name="example3" time="0.002"/>
  <testcase classname="com.test.Test1" name="example4" time="0.004">
    <failure type="java.lang.AssertionError" message="This test should be failed">
      <![CDATA[java.lang.AssertionError: This test should be failed
at com.test.Test1.example4(Test1.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- example4 -->
  <testcase classname="com.test.Test1" name="skippedByDependencyTest" time="0.0">
    <skipped/>
  </testcase> <!-- skippedByDependencyTest -->
  <testcase classname="com.test.Test2" name="test1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at com.test.Test2.test1(Test2.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="com.test.Test2" name="test2" time="0.002"/>
  <testcase classname="com.test.Test2" name="test3" time="0.003"/>
  <testcase classname="com.test.Test2" name="test4" time="0.002">
    <failure type="java.lang.AssertionError" message="This test should be failed">
      <![CDATA[java.lang.AssertionError: This test should be failed
at com.test.Test2.test4(Test2.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase classname="com.test.Test2" name="skippedByDependencyTest" time="0.0">
    <skipped/>
  </testcase> <!-- skippedByDependencyTest -->
</testsuite> <!-- User Tests -->
